src/Functions/FunctionUnaryArithmetic.h

```diff
@@ -130,6 +132,46 @@ struct FixedStringUnaryOperationImpl
     }
 };
 
+template <typename Op>
+struct FixedStringUnaryOperationReduceImpl
```

Comment:
Now it should be renamed to `StringUnaryOperationReduceImpl`, as nothing "Fixed" remains.